assistant.go

```diff
@@ -0,0 +1,160 @@
+package slack
+
+import (
+	"context"
+	"encoding/json"
+	"net/url"
+)
+
+// AssistantThreadSetStatusParameters are the parameters for AssistantThreadSetStatus
+type AssistantThreadsSetStatusParameters struct {
+	ChannelID string `json:"channel_id"`
+	Status    string `json:"status"`
+	ThreadTS  string `json:"thread_ts"`
+}
+
+// AssistantThreadSetTitleParameters are the parameters for AssistantThreadSetTitle
+type AssistantThreadsSetTitleParameters struct {
+	ChannelID string `json:"channel_id"`
+	ThreadTS  string `json:"thread_ts"`
+	Title     string `json:"title"`
+}
+
+// AssistantThreadSetSuggestedPromptsParameters are the parameters for AssistantThreadSetSuggestedPrompts
+type AssistantThreadsSetSuggestedPromptsParameters struct {
```

Comment:
Could you please add the `title` optional argument? I know the [documentation](https://api.slack.com/methods/assistant.threads.setSuggestedPrompts#arg_title) says its currently ignored by the Slack client, but I'd like for this to match the API spec.